changelogs/fragments/1392-proposed_fixes_for_backup_behavior_1383.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - vmware_cfg_backup - Check if hostname is an ESXi host or a vCenter appliance, discard esxi_hostname if target is an ESXi host (https://github.com/ansible-collections/community.vmware/pull/1392).
```

```suggestion
  - vmware_cfg_backup - Fix a possible urlopen error when connecting directly to an ESXi host (https://github.com/ansible-collections/community.vmware/issues/1383).
```

I think this might be too technical, but doesn't really describe what's fixed. What do you think about my suggestion?

---

You're correct. I tend to favor technical descriptions over "human" descriptions everywhere :)